libcxx/include/__coroutine/coroutine_handle.h

```diff
@@ -55,21 +55,21 @@ struct _LIBCPP_TEMPLATE_VIS coroutine_handle<void> {
   _LIBCPP_HIDE_FROM_ABI constexpr explicit operator bool() const noexcept { return __handle_ != nullptr; }
 
   _LIBCPP_HIDE_FROM_ABI bool done() const {
-    _LIBCPP_ASSERT_UNCATEGORIZED(__is_suspended(), "done() can be called only on suspended coroutines");
+    _LIBCPP_ASSERT_VALID_EXTERNAL_API_CALL(__is_suspended(), "done() can be called only on suspended coroutines");
```

Comment:
I agree it might look a bit odd, but you could call it an external API. I don't mind to keep it in the same category, but I would like to specifically mention that compiler intrinsics are part of this category.

Reply:
I agree, I think I would mention that we consider anything external to the library (including compiler builtins) as an "external API" for the purpose of this category.

Reply:
Good points, done.